content/articles/an-introduction-to-defi/index.md

```diff
@@ -0,0 +1,129 @@
+
+#### Bridge
+A blockchain is a distributed database with Atomic settlement. As a result, the blockchain may need to interact with another blockchain that may have another type of financial asset. To ensure the successful interaction between the two blockchains, a bridge enables the transfer of such information.
+
+### The Layers in The DeFi Stack
+There are five layers in the DeFi Stack. They are as follows:
+- The Settlement Layer
+- The Asset Layer
+- The Protocol Layer
+- The Application Layer
+- The Aggregation Layer
+
+#### The Settlement Layer
+This is the first layer of the DeFi Stack upon which other DeFi transactions are logged on the blockchain. On this layer, transactions occurring on DeFi applications are settled through the use of a currency which could be a stablecoin or a cryptocurrency such as USDT and ETH.
+
+#### The Asset Layer
+The asset layer implements the different financial assets, such as the native protocol asset (Ethereum). In this layer, there are standard protocols on how to interact with the token contract. ERC20 is one of the standards used on this layer to maintain a different balance for the asset. 
+
+#### The Protocol Layer
+The protocols are created as smart contracts with rules defined to govern tasks and activities on the asset. The protocol layer is where different types of financial services are implemented such as Lending, Derivatives, Asset Management, and more. As such, it provides liquidity to the DeFi ecosystem.
+
+#### The Application Layer
+The application layer is user-facing. It is an intermediary between the user and the cloud servers that interpolate with the entire stacks of the blockchain and particularly, the underlying protocol layer.
+
+#### The Aggregation Layer
+This layer consists of different applications from the application layer all stacked up into one application to provide user access to other DeFi services. An example is a cryptocurrency wallet with multiple services such as staking, lending and exchange.
+
+### DeFI Building Block and Services
+The following are some of the building blocks and services of Decentralized Finance:
+- Asset Tokenization.
+- Stablecoin.
+- Decentralized Exchanges. 
+
+#### Asset Tokenization
+Tokenization is the process of securitizing an asset into the blockchain network. Upon tokenization of such security, the representation of the asset on the blockchain is what we call “Tokens”. 
+ 
+Asset tokenization can be in the form of a Governance Token whereby the future dimensioned pathway of the token is decided by a vote on the blockchain. A security token is also another form of asset tokenization whereby the token is used to represent the state of a real-life asset.
+
+#### Stablecoin
+These are tokens whose price is tagged to the type of assets such as fiat and precious metal. Stable coins can be created via Off-chain Collateral(Fiat), OnChain Collateral (Crypto Asset), Non-Collateralized (Algorithmic).
```

Comment:
Also, add external links to each of these: Off-chain Collateral (Fiat), OnChain Collateral (Crypto Asset), Non-Collateralized (Algorithmic)